CHANGELOG.md

```diff
@@ -18,14 +18,18 @@ Given the `MAJOR.MINOR.PATCH` pattern, here is how we decide to increment:
 ## Added
 
 - Frontend: Added task for generating JavaScript code docs with `gulp docs`.
+- Browser tests for most of the organisms generated by Wagtail
+- `cfgov/scripts/_atomic_helpers.py` to provide streamfield data
 
 ### Changes
 - Use bare value of RichText field if value type is not RichText
 - Check against Activity Log topics when generating View More link
+- initial_test_data script now uses streamfield block data from a new file called _atomic_helpers.py.
 
 ### Removed
 - Event RSVP email link button
 - `atomicName` parameter from `checkDom` atomic helper.
+- PostPreview organism since it was unused
```

Comment:
Mind clarifying it's only removed `from streamfield block choices` so no one thinks it's removed from the jinja2 or styles directories.